### DISCUSSION TITLE
start in quake mode

### DISCUSSION THREAD

**Discussion by jassde** [microsoft/terminal#15013] [Ideas]
Start the terminal in quake mode and if you need a normal window to appear, hotkey windows and down key to change the quake window into a normal window for example

**Comment by jameswylde** [comment]
Whilst you wait for one of the Terminal folks to chime in on this - I'll add the below which can offer you some of that functionality, whilst maybe not in the cleanest way.

- Create a shortcut, and in the address/location add `wt.exe -w _quake` - this can replace your normal Terminal shortcut so that when launched every new session is in quake mode by default
- In _Actions_ amend the _New Window_ shortcut to be your preferred binding (however believe your request of Win+Down won't be possible due to snapping shortcuts) - so when in the quake window as normal, this binding will open a standard terminal window. (The quake window won't close automatically though)
